### PR TITLE
360 gfw fires

### PR DESCRIPTION
**Pull request by KaylaKremer**
https://github.com/wri/gfw-mapbuilder/issues/360

### CODE REVIEW

**Comment by KaylaKremer** [comment]
@SwampGuzzler Should we go ahead and merge in `resources.js` and an updated `external.pug` as well since the default URL for VIIRS and MODIS has changed?

File: src/js/components/LayerPanel/FiresControls.js [hunk 1]
```diff
@@ -18,9 +15,23 @@ export default class FiresControls extends React.Component {
...
 
-    this.min = props.layer.id === 'VIIRS_ACTIVE_FIRES' ? moment(new Date('2016', 0, 8)) : moment(new Date('2012', 0, 1));
+    const currentDate = new Date();
+    const oneYearAgo = currentDate.setFullYear(currentDate.getFullYear() - 1);
     const max = new Date();
+
+    this.min = moment(oneYearAgo);
     this.max = moment(max);
+    this.fireOptions = [
+      {label: '24HR', value: '0'},
```
Comment (christianhoward) [inline src/js/components/LayerPanel/FiresControls.js]: Lets change `label` to "Past 24 hours", "Past 48 hours" ...

File: src/js/helpers/LayersHelper.js [hunk 2]
```diff
@@ -32,15 +33,62 @@ const LayersHelper = {
...
+  updateFiresLayerDefinitions (startDate = null, endDate = null, layer, selectValue = null, dontRefresh) {
     if (brApp.map) {
-      const queryString = this.generateFiresQuery(startDate, endDate);
       const firesLayer = layer.hasOwnProperty('visibleLayers') ? layer : brApp.map.getLayer(layer.id);
-      const defs = [];
-
-      if (firesLayer) {
-        firesLayer.visibleLayers.forEach(val => { defs[val] = queryString; });
-        firesLayer.setLayerDefinitions(defs, dontRefresh);
+      console.log('fires layer', firesLayer);
```
Comment (christianhoward) [inline src/js/helpers/LayersHelper.js]: Remove console log

**Comment by christianhoward** [comment]
I approve Kayla's merge!

**Pull request merged**